### PR TITLE
fix(tixati): adapt to changed website structure

### LINKED ISSUE #1225: Tixati fails because of website structure changes
The website has been restructured slightly so we need to adapt

### PR DESCRIPTION
**Pull request by philclifford**
fixes #1225

### CODE REVIEW

**Review by flexiondotorg — APPROVED** [review]

File: 01-main/packages/tixati [hunk 1]
```diff
@@ -1,7 +1,7 @@
 DEFVER=1
-get_website "https://www.tixati.com/download/linux.html"
+get_website "https://www.tixati.com/linux"
 if [ "${ACTION}" != "prettylist" ]; then
-    URL="$(grep "amd64\.deb\"" "${CACHE_FILE}" | head -n1 | cut -d'"' -f2)"
```
Comment (rasa) [inline 01-main/packages/tixati:4]: @philclifford why move away from `"${CACHE_FILE}"`?

File: 01-main/packages/tixati [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+    URL=$(grep -m 1 -o  '\http.*amd64\.deb.asc'  /var/cache/deb-get/tixati.html  | sed 's/.asc//')
```
Comment (rasa) [inline 01-main/packages/tixati:4]: It pedantic, but I think this wants to be
`URL=$(grep -m 1 -o 'http.*amd64\.deb\.asc'  /var/cache/deb-get/tixati.html  | sed 's/\.asc//')`
to escape the two periods.
The backslash before the h is a typo, yes?

Comment (philclifford) [reply 01-main/packages/tixati]: yes (it was the one for .asc but this laptop old likes to move cursors around for fun and I wasn't looking where I was typing clearly :-) 
Pedantic is my middle name :-)

File: 01-main/packages/tixati (hunk 1 above)
Comment (philclifford) [reply 01-main/packages/tixati]: Unintended consequence of tired and hurried attempt to rapidly fix what I broke while also fixing breakages from upstream.   
Sorry.

**Pull request merged**